From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: xiaoyinl <xiaoyinl@users.noreply.github.com>
Date: Sat, 18 Oct 2025 11:07:42 +0800
Subject: [PATCH] 8575: Correct winsock.h headers in
 CreateExampleGDTArchiveScript.java

There is a compile error if you include both winsock.h and winsock2.h. It says you should only include winsock2.h.
---
 .../Base/ghidra_scripts/CreateExampleGDTArchiveScript.java      | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Ghidra/Features/Base/ghidra_scripts/CreateExampleGDTArchiveScript.java b/Ghidra/Features/Base/ghidra_scripts/CreateExampleGDTArchiveScript.java
index 6823a5daab..60de9e4c4b 100644
--- a/Ghidra/Features/Base/ghidra_scripts/CreateExampleGDTArchiveScript.java
+++ b/Ghidra/Features/Base/ghidra_scripts/CreateExampleGDTArchiveScript.java
@@ -252,8 +252,6 @@ public class CreateExampleGDTArchiveScript extends GhidraScript {
 				"af_irda.h",
 				"in6addr.h",
 				"mstcpip.h",
-				"ws2def.h",
-				"winsock.h",
 				"winsock2.h",
 				"nsemail.h",
 				"nspapi.h",
-- 
2.45.1

